src/Core/Import/EntityField/Provider/AddressFieldsProvider.php

```diff
@@ -0,0 +1,95 @@
+<?php
+/**
+ * 2007-2018 PrestaShop.
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\PrestaShop\Core\Import\EntityField\Provider;
+
+use PrestaShop\PrestaShop\Core\Import\EntityField\EntityField;
+use PrestaShop\PrestaShop\Core\Import\EntityField\EntityFieldCollection;
+use Symfony\Component\Translation\TranslatorInterface;
+
+/**
+ * Class AddressFieldsProvider defines an address fields provider.
+ */
+final class AddressFieldsProvider implements EntityFieldsProviderInterface
+{
+    /**
+     * @var TranslatorInterface
+     */
+    private $translator;
+
+    /**
+     * @param TranslatorInterface $translator
+     */
+    public function __construct(TranslatorInterface $translator)
+    {
+        $this->translator = $translator;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getCollection()
+    {
+        $fields = [
+            new EntityField('id', $this->trans('ID', 'Admin.Global')),
+            new EntityField('alias', $this->trans('Alias', 'Admin.Shopparameters.Feature'), '', true),
+            new EntityField('active', $this->trans('Active  (0/1)')),
```

Comment:
No domains or default domains indicated here?

Reply:
Default domain for these messages is 'Admin.Advparameters.Feature', it's defined in trans() method in the same class. It's a custom private method, so I've set default domain there. :)

Reply:
Yes exactly, it's at line 91. Btw, I've just checked these texts in a different language and they are being translated properly :)